### ISSUE TITLE
Design Feedback - Design Intent - Authenticated Experience, My VA Enhancements - Payment

### ISSUE THREAD

**Issue by seanasewell** [department-of-veterans-affairs/va.gov-team#34047]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
- The active links with colored (yellow, blue, and gray) backgrounds are not an approved link style. Remove the background color. Anything new will need to go through the experimental design process.

**Should:**
- Consider changing error "!" icon in front of messages and payment sections to warning "!" icon. Another option could be to remove the icon all together since warning icons typically insinuate something has gone wrong. The warning icon seems applicable for payments that need made however will messages always be in negative consequence of something? See design system alert box section for documentation around this. https://design.va.gov/components/alertboxes#warning-alert

**Consider:**

**Comment by hdjustice** [comment]
@Samara-Strauss This is the ticket resulting from the Design Intent meeting.

**Comment by Samara-Strauss** [comment]
> The active links with colored (yellow, blue, and gray) backgrounds are not an approved link style. Remove the background color. Anything new will need to go through the experimental design process.

@seanasewell Is there any leeway on this? Making the background white or gray is likely going to make these action items less noticeable, and the whole point of these callouts is to alert veterans that they need to take an action. We're using color here to communicate the following visual hierarchy:

1. Action items (yellow background)
2. Status information (grey boxes)
3. Links (white background)

We've heard repeatedly in research that action items are the most important thing we can call out to veterans on the My VA dashboard (I can provide links to reports if you want to see them), so having these important action items blend in with other status information or links would likely make the design less effective.

TL;DR Having effective visual callout is essential to highlighting action items. What can we do to move the design forward and get something approved quickly by the design system team? We're happy to do whatever testing is necessary to move this forward and prove out our hypothesis (or disprove it and let the issue go). Just let us know what steps you recommend.

**Comment by allison0034** [comment]
@Samara-Strauss, unfortunately we do not have any approved active link styles that currently have background colors. The styles used in the Mural are [background alerts](https://design.va.gov/components/alertboxes#background-color-only-alert) which are for immediate feedback. If you would like to propose a new style it will need to go through the [experimental design process](https://design.va.gov/experimental-design/).

**Comment by Samara-Strauss** [comment]
Great, we will kick off the experimental design process.

**Comment by tressaellen** [comment]
We've kicked off the [experimental design process with this ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/606) - it's set to be discussed 1/7. 

I've updated the icons as well. [The latest designs can be seen in this mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=tressafurner7646). Let me know if there's anything else you need from me! Thanks!
